tests/codegen/call_offsetof.cpp

```diff
@@ -0,0 +1,16 @@
+#include "common.h"
+
+namespace bpftrace {
+namespace test {
+namespace codegen {
+
+TEST(codegen, call_offsetof)
+{
+  test("struct Foo { int x; long l; char c; } BEGIN { @x = offsetof(c, struct Foo) }",
```

Comment:
This is a bit tricky as the test may fail on some arches due to different ABI. Since we don't have a way to filter codegen tests per-arch, I'd suggest testing the offset of `x` which will be 0 pretty much everywhere.